src/app/inventory/store/season.ts

```diff
@@ -14,30 +16,59 @@ import { DimItem } from '../item-types';
 
 const SourceToD2Season: Record<number, number> = D2SeasonFromSource.sources;
 
-export function getSeason(item: DimItem): number {
-  if (item.classified) {
+export function getSeason(
+  item: DimItem | DestinyInventoryItemDefinition,
+  defs?: D2ManifestDefinitions
+): number {
```

Comment:
This is awful but the rest of DIM wants to derive this info based on DimItem while this one needs to be based on defs. So this essentially duplicates some bits of makeItem/makeFakeItem... Suggestions welcome